app/models.py

```diff
@@ -60,12 +60,12 @@ class CFAUser(models.Model):
     cc_emails = models.ManyToManyField("CCEmail", blank=True)
     mission_statement = models.TextField(blank=True)
 
-    def __unicode__(self):
+    def __str__(self):
         if self.is_funder:
             # some funders might not have funder_names
-            return unicode(self.funder_name) or unicode(self.user)
+            return str(self.funder_name) or str(self.user)
         else:
-            return unicode(self.user)
+            return str(self.user)
```

Comment:
Does this work in Python 2 with unicode/special characters? It doesn't need to, I'm just wondering.

Reply:
This doesn't work with Python 2, unfortunately. I could keep in both __unicode__ and __str__, but this seemed repetitive to me.